tools/snapshot/snapshot_builder.cc

```diff
@@ -99,7 +99,7 @@ std::string SnapshotBuilder::Generate(
     // creator is still alive.
     main_instance->Dispose();
     result = FormatBlob(&blob, isolate_data_indexes);
-    delete blob.data;
+    delete [] blob.data;
```

Comment:
I think we predominantly (but not exclusively) write this as `delete[]` (no space.)

There's a s/mk_snapshot/mksnapshot/ typo in the commit log but otherwise LGTM.

Reply:
ah yes since make lint did not raise any eyebrow I did not spot ok will change :)